.github/workflows/trivy-docker.yaml

```diff
@@ -25,15 +25,15 @@ on:
   workflow_dispatch:
 
 permissions:
-  actions: none
+  actions: write
   checks: none
   contents: read
   deployments: none
   issues: none
   packages: none
   pull-requests: none
   repository-projects: none
-  security-events: none
+  security-events: write
```

Comment:
I think this is the only one needed, it's documented here: https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/uploading-a-sarif-file-to-github#example-workflow-for-sarif-files-generated-outside-of-a-repository

Reply:
Ahhh nice find!